e.cpp

```diff
@@ -0,0 +1,13 @@
+#include <iostream>
+using namespace std;
+
+int main()
+{
+    while(true)
```

Comment:
ee `'e' == 'e'` eeeee eee!

```suggestion
    while('e' == 'e')
```

Reply:
fc176ad1fce16a11ef272afcd523bd73c5a0ebc1
eeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee